chains/evm/listener/listener.go

```diff
@@ -89,27 +95,33 @@ func (l *EVMListener) ListenToEvents(ctx context.Context, startBlock *big.Int, m
 			// Sleep if the difference is less than needed block confirmations; (latest - current) < BlockDelay
 			if new(big.Int).Sub(head, endBlock).Cmp(l.blockConfirmations) == -1 {
 				time.Sleep(l.blockRetryInterval)
+				span.AddEvent("The block difference is too low")
+				span.SetStatus(codes.Ok, "The block difference is too low")
```

Comment:
Do we need both here (event and status)?

Reply:
Yeah for now let it be, also DD is not showing descriptions for Status, only events. In general, I want to investigate Jaeger or SigNoz or other OS tracing observers, bcs I am not totally like DD.